src/Api/AdminConsole/Controllers/OrganizationsController.cs

```diff
@@ -899,8 +899,9 @@ public async Task EnableCollectionEnhancements(Guid id)
         var orgUsers = await _organizationUserRepository.GetManyByOrganizationAsync(id, null);
         await Task.WhenAll(orgUsers
             .Where(ou => ou.UserId.HasValue &&
+                         ou.Status == OrganizationUserStatusType.Confirmed &&
```

Comment:
Not strictly related to the bug, but I thought there's no point syncing users who aren't confirmed as they won't have access to any org data anyway.